__tests__/task6-get-api-articles-articleid-comments.test.js

```diff
@@ -0,0 +1,79 @@
+const { app, request, db, seed, data } = require("../testIndex");
+require("jest-sorted"); // https://www.npmjs.com/package/jest-sorted
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("should return a 200 status code, indicating the endpoint is accessible", () => {
+    return request(app).get("/api/articles/1/comments").expect(200);
+  });
+
+  test("should return back as an array", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .then(({ body }) => {
+        expect(Array.isArray(body)).toBe(true);
+      });
+  });
+
+  test("should return the correct key and data type for value", () => {
```

Comment:
should test for 200 'returns an empty array for article with no comments' to be sure your function to check an article exists works.